README.md

```diff
@@ -160,7 +160,7 @@ There are other Git GUI clients out there such as [SmartGit](http://www.syntevo.
 
 ### Clone the GitHub Repository
 
-1. Form the command line...
+1. From the command line...
```

Comment:
Just to show you what it looks like, here is a comment inline in case you want to make a comment on a specific line of code.